client/src/modules/journal/modals/comment.modal.html

```diff
@@ -0,0 +1,33 @@
+<form name="ModalForm"
```

Comment:
It would be really cool if this would use the exact same modal HTML as the account statement.  Do you think that is possible?

Reply:
I think it can be a litle bit complicated because there is a submit function on the button click, this function has the API url witch is not the same for both (Account statement and posting journal)

Reply:
The JS can be different - just bind a different controller.  That way, the `submit()` method will be different, but the HTML will be the same.  But the HTML can be reused from [comment.modal.html](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/account_statement/modals/comment.modal.html).

This way we don't have to maintain two HTML files and fix bugs in two different places :+1:  It also respects the principle of [Don't Repeat Yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).